src/main/java/step4/domain/Position.java

```diff
@@ -0,0 +1,22 @@
+package step4.domain;
+
+public class Position {
+
+    int position;
+
+    public Position() {
+        this(0);
+    }
+
+    public Position(int position) {
+        this.position = position;
+    }
+
+    public int getPosition() {
+        return position;
+    }
```

Comment:
```suggestion
    public int position() {
        return position;
    }
```

포비님 영상을 보시면 아시겠지만, 만약 엘레강트 오브젝트에 욕심이 있으시다면
builder 메서드는 명사형으로 지어주셔도 괜찮을 것 같습니다.
(다만 저는 엘레강트 오브젝트에 관련해서는 지나치게 집착을 하고 싶지 않습니다. 메서드명이 이해가 간다면 그냥 넘어갈려고 합니다.)
굳이 수정하지 않으셔도 괜찮습니다.